textrepo-app/src/main/java/nl/knaw/huc/FlywayConfig.java

```diff
@@ -0,0 +1,12 @@
+package nl.knaw.huc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class FlywayConfig {
+
+  @JsonProperty
+  public boolean cleanDisabled;
```

Comment:
Kunnen we hier een 'double negative' weghalen door het 'clean' te noemen? Ipv "cleanDisable=false" als je wél 'clean' wil, kun je dan 'clean=true' zetten?

Reply:
Dat is de term zoals flyway het gebruikt, voor de duidelijkheid leek het me fijner om die terminologie aan te houden?

Reply:
Laten we het dan maar laten zoals het is. Ik heb de documentatie bij Flyway gelezen en dan snap ik nog niet wat 'make clean --disableClean' doet ;-)